src/frontend/packages/suse-theme/package.json

```diff
@@ -1,17 +1,19 @@
 {
-  "name": "@suse/theme",
-  "version": "0.0.1",
+  "name": "@susestratos/theme",
+  "version": "1.0.0",
```

Comment:
Should suse-extensions also be 1.0.0 for the initial release?

Reply:
0.0.0 as a placeholder